chordspy/.github/workflows/release.yaml

```
@@ -0,0 +1,167 @@
+    runs-on: ubuntu-latest
+    environment:
+      name: release
+    steps:
+      - name: Dispatch Repository Dispatch event
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.PYPI_API_TOKEN }}
+          repository: ${{ env.OWNER }}/{{ env.TAP_NAME }}
+          event-type: "update-formula"
+          client-payload: |-
+            { 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)